qpc/server/commands.py

```diff
@@ -15,3 +15,4 @@
 from qpc.server.configure_host import ConfigureHostCommand
 from qpc.server.login_host import LoginHostCommand
 from qpc.server.logout_host import LogoutHostCommand
```

Comment:
Copyright should be updated to have 2017-2018 date.

Reply:
fixed 👍